lib/src/error.rs

```diff
@@ -238,6 +238,10 @@ pub enum HandleError {
     #[error("Invalid pending request handle: {0}")]
     InvalidPendingRequestHandle(crate::wiggle_abi::types::PendingRequestHandle),
 
+    /// A lookup handle was not valid.
+    #[error("Invalid pending lookup handle: {0}")]
```

Comment:
Should we add "Object Store" into the error message? It might help people know what the lookup handle is in relation to

Reply:
That sounds smart, I think I might do a lot of `s/lookup/object/g`, or I suppose now `value` or `kv`. `PendingKVRequest` `PendingKVRequestHandle`???

Reply:
yes, kv for sure 👍

Reply:
Done :+1: